workitem/link/link_repository.go

```diff
@@ -111,6 +113,10 @@ func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targe
 		}
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
+	// save a revision of the created work item link
+	if err := r.revisionRepo.Create(ctx, creatorID, RevisionTypeCreate, *link); err != nil {
+		return nil, errs.WithStack(err)
```

Comment:
@aslakknutsen @kwk my point is: `r.revisionRepo.Create()` already returns the following error if something wrong happened:
````
return errors.NewInternalError(fmt.Sprintf("failed to create new work item link revision: %s", err.Error()))
````

I'm fine with wrapping the error once the call returned, but I'm not sure what error message to add here, honestly: the creation failed, as it has been reported by the called method.